### PR TITLE
Complete html-links exercise

### PR DESCRIPTION
**Pull request by BenGrimmius**
[Screen Recording 2022-08-24 at 7.52.14 PM.mov.zip](https://github.com/BenGrimmius/mod1-0822-code-solutions/files/9420953/Screen.Recording.2022-08-24.at.7.52.14.PM.mov.zip)

### CODE REVIEW

File: html-links/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Links</title>
+</head>
+<body>
+  <ul>
```
Comment (scrumptiousdev) [inline html-links/index.html:11]: Woohoo! Great job! Just one small request: I think we are missing heading for this page!

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

**Comment by BenGrimmius** [comment]
Oh my bad. Alright I added the <h1> element

**Review by scrumptiousdev — APPROVED** [review]
🎸 Approved!

**Comment by scrumptiousdev** [comment]
> Oh my bad. Alright I added the
> 
> # element

# Nicely done, sir!

**Pull request merged**